spec/hashTable.js

```diff
@@ -11,19 +11,69 @@ describe('hashTable', () => {
 
   context('put()', () => {
     it('adds a key-value pair to the hash table.', () => {
-<<<<<<< Updated upstream
-      const myHash = new HashTable()
+      const ht = new HashTable()
+      ht.put('name', 'frankenstein')
+      expect(ht.size()).to.equal(1)
```

Comment:
Thanks I see my approach was too bunched up and I'll definitely correct that. Thank you